### PR TITLE
kubetest: build windows and linux artifcats with bazel

### PR DESCRIPTION
**Pull request by yujuhong**
I didn't copy the artifacts out because all the release tars seem to be built properly after running the two bazel commands back-to-back on my desktop.

@ixdy @BenTheElder please take a look and see if this change seems reasonable to you. Thanks!

### CODE REVIEW

**Comment by BenTheElder** [comment]
/unassign @ixdy @BenTheElder

You'll need a current approver.

File: kubetest/build.go [hunk 1]
```diff
@@ -85,10 +85,29 @@ func (b *buildStrategy) Build() error {
 		target = "quick-release"
 	case "release":
 		target = "release"
+	case "gce-windows-bazel":
+		// bazel doesn't support building multiple platforms yet.
```
Comment (BenTheElder) [inline kubetest/build.go]: multiple platforms _simultaneously_

**Comment by BenTheElder** [comment]
does this work? do you need to copy in between?

overall looks relatively fine

File: kubetest/build.go (hunk 1 above)
Comment (yujuhong) [reply kubetest/build.go]: Done

**Comment by yujuhong** [comment]
> does this work? do you need to copy in between?

I ran the bazel build commands on my desktop and checked the artifacts. They were all there.

I'm not sure what more I need to do at this point :-|

**Comment by yujuhong** [comment]
/assign @spiffxp 
/assign @Katharine 

This PR only adds a new option with relatively straightforward code.  PTAL, thanks!

File: kubetest/build.go [hunk 2]
```diff
@@ -85,10 +85,29 @@ func (b *buildStrategy) Build() error {
...
+	case "gce-windows-bazel":
+		// bazel doesn't support building multiple platforms simultaneously
+		// yet. We add custom logic here to build both Windows and Linux
+		// release tars. https://github.com/kubernetes/kubernetes/issues/76470
+		// TODO: remove this after bazel supports the feature.
 	default:
 		return fmt.Errorf("Unknown build strategy: %v", b)
 	}
 
+	if target == "gce-windows-bazel" {
```
Comment (spiffxp) [inline kubetest/build.go]: It seems like you mean to either set `target` above, or use `*b` here.  As written I don't understand how this block would get executed

Comment (yujuhong) [reply kubetest/build.go]: Yep, you're right. I wanted to use *b directly and forgot to change. Fixed.

**Pull request merged**